wasm2c/README.md

```diff
@@ -73,23 +80,31 @@ int main(int argc, char** argv) {
   /* Initialize the Wasm runtime. */
   wasm_rt_init();
 
-  /* Initialize the fac module. */
-  Z_fac_init();
+  /* Initialize the `fac` module (this registers the module's function types in
+   * a global data structure) */
+  Z_fac_init_module();
+
+  /* Declare an instance of the `fac` module. */
+  Z_fac_module_instance_t module_instance;
```

Comment:
e.g. what if this was just `Z_fac_instance_t instance;`  ?

Reply:
Agreed (done).